tests/models/bert/test_bert.py

```diff
@@ -59,13 +59,13 @@ def decode_output(outputs):
         answer_before = decode_output(outputs_before)
 
         # Compile model with ttnn backend
-        option = torch_ttnn.TorchTtnnOption(device=self.device)
+        option = torch_ttnn.TorchTtnnOption(device=self.device, metrics_path="BERT")
         m = torch.compile(m, backend=torch_ttnn.backend, options=option)
 
         # Run inference with the compiled model
         with torch.no_grad():
-            outputs_after = m(**inputs)
-        option._out_fx_graphs[0].print_tabular()
+            outputs_after = RunTimeMetrics("BERT", "compiled", lambda: m(**inputs))
+        # option._out_fx_graphs[0].print_tabular()
```

Comment:
remove?

Reply:
Should we actually keep this? This prints out the table view of the converted graph.

Reply:
We can keep it for now if it is useful